### PR TITLE
Reworked a bit of bounty stuff.

### PR DESCRIPTION
**Pull request by Owai-Seek**
## About The Pull Request

I don't play cargo, but I do play service roles and assistant. I find it a constant hassle to have assistant and chef bounties that require botany or higher-access (or access to limited-amount items) to complete.

This makes chef bounties more accessible for a roundstart chef, without the need of botany. As "tradeoff" there is a new category for bounties that "need" both a chef, and botany to pull it off. (Let's be honest, it mainly needs botany, chef is a horrible role.)

This moves several "silly" bounties out of assistant category and makes a new bounty category for them.

## Why It's Good For The Game

More bounties are good. My pricing might be a bit off, so maybe change that a bit?

## Changelog
:cl:
add: "silly" bounties
add: "gardenchef" bounties
del: several bounties that require seriously good RNG to pull off.
tweak: moved several chef and assistant bounties to silly and gardenchef
balance: modified several bounty point rewards
server: added new files "silly.dm" and "gardenchef.dm"
/:cl:

### CODE REVIEW

File: code/modules/cargo/bounties/silly.dm [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	name = "Mouse Traps"
+	description = "Some shoeless hoodlum is intentionally tracking paint all over Centcom. Send us some mouse traps so we can capture the culprit."
+	reward = 1200
+	required_count = 12
+	wanted_types = list(/obj/item/assembly/mousetrap)
+
+/datum/bounty/item/silly/action_figures
+	name = "Action Figures"
+	description = "The vice president's son saw an ad for action figures on the telescreen and now he won't shut up about them. Ship some to ease his complaints."
+	reward = 4000
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/bounties/silly.dm]: Way to high for 300 plastic 2500 would be better

File: code/modules/cargo/bounties/silly.dm [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	required_count = 5
+	wanted_types = list(/obj/item/toy/figure)
+
+/datum/bounty/item/silly/clown_box
+	name = "Clown Box"
+	description = "The universe needs laughter. Stamp cardboard with a clown stamp and ship it out."
+	reward = 1000
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/bounties/silly.dm:25]: 800 is p-good seeings how its 1 sheet of cardboard to make this

File: code/modules/cargo/bounties/assistant.dm [hunk 3]
```diff
@@ -76,44 +70,45 @@
...
+	required_count = 3
 	wanted_types = list(/obj/item/storage/briefcase, /obj/item/storage/secure/briefcase)
 
 /datum/bounty/item/assistant/sunglasses
 	name = "Sunglasses"
-	description = "A famous blues duo is passing through the sector, but they've lost their shades and they can't perform. Ship new sunglasses to CentCom to rectify this."
-	reward = 1000
-	required_count = 2
+	description = "A famous bounty hunter is passing through the sector, and they've lost their shades. They're too embarrassed to hunt criminals with their naked eyes. Ship new sunglasses to CentCom to rectify this."
+	reward = 1750
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/bounties/assistant.dm]: Lowering this to 1000 is a good idea, its just one pare of glasses and they are easy to come by. I.e you can spawn with them...

File: code/modules/cargo/bounties/assistant.dm [hunk 4]
```diff
@@ -26,8 +26,8 @@
 /datum/bounty/item/assistant/soap
 	name = "Soap"
 	description = "Soap has gone missing from CentCom's bathrooms and nobody knows who took it. Replace it and be the hero CentCom needs."
-	reward = 1000
-	required_count = 3
+	reward = 1200
+	required_count = 3 //You can (apparently) get soap from the mining rewards vendor.
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/bounties/assistant.dm:26]: You can make soap out of rats

File: code/modules/cargo/bounties/assistant.dm [hunk 5]
```diff
@@ -1,20 +1,20 @@
 /datum/bounty/item/assistant/strange_object
 	name = "Strange Object"
 	description = "Nanotrasen has taken an interest in strange objects. Find one in maint, and ship it off to CentCom right away."
-	reward = 600
+	reward = 2200
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/bounties/assistant.dm]: 1000 at most, these things spawn all over the place and should not be worth as much as taser crates

**Review by Trilbyspaceclone — CHANGES_REQUESTED** [review]
Some things are a bit high some are a bit low, -  Bolas and Metal shields should be upped a bit imo

File: code/modules/cargo/bounties/silly.dm (hunk 1 above)
Comment (Owai-Seek) [reply code/modules/cargo/bounties/silly.dm]: The reason it's so high is because you need to have they toy lathe researched (except on Delta) in order to complete it. This takes a significant amount of time and resources. This was the default number when I moved it from Assistant Bounties to Silly Bounties.

File: code/modules/cargo/bounties/assistant.dm (hunk 3 above)
Comment (Owai-Seek) [reply code/modules/cargo/bounties/assistant.dm]: Only certain jobs spawn with sunglasses, and they're a fairly powerful item to give up for a bounty. (Flash Immunity with no vision impairment) My concern was mostly reducing the count, as getting sunglasses as an assistant can be a pain.

File: code/modules/cargo/bounties/assistant.dm [hunk 6]
```diff
@@ -76,44 +70,45 @@
...
+	reward = 1200
+	required_count = 3
 	wanted_types = list(/obj/item/storage/briefcase, /obj/item/storage/secure/briefcase)
 
 /datum/bounty/item/assistant/sunglasses
 	name = "Sunglasses"
-	description = "A famous blues duo is passing through the sector, but they've lost their shades and they can't perform. Ship new sunglasses to CentCom to rectify this."
-	reward = 1000
-	required_count = 2
+	description = "A famous bounty hunter is passing through the sector, and they've lost their shades. They're too embarrassed to hunt criminals with their naked eyes. Ship new sunglasses to CentCom to rectify this."
```
Comment (Ghommie) [inline code/modules/cargo/bounties/assistant.dm]: Don't remove that Blues Brothers reference, please!

File: code/modules/cargo/bounties/assistant.dm [hunk 7]
```diff
@@ -152,14 +147,14 @@
...
+// /datum/bounty/item/assistant/earmuffs
+// 	name = "Earmuffs"
+// 	description = "Central Command is getting tired of your station's messages. They've ordered that you ship some earmuffs to lessen the annoyance."
+// 	reward = 1000
+// 	wanted_types = list(/obj/item/clothing/ears/earmuffs)
 
-/datum/bounty/item/assistant/handcuffs
-	name = "Handcuffs"
+/datum/bounty/item/assistant/cuffs
+	name = "Cuffs"
```
Comment (Ghommie) [inline code/modules/cargo/bounties/assistant.dm]: Undo this change, I don't see a reason why an abbreviation should be used here.

**Review by silicons — CHANGES_REQUESTED** [review]
what ghom said

File: code/modules/cargo/bounties/assistant.dm (hunk 3 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/bounties/assistant.dm]: You spawn in loadout any job with sunglasses

Comment (Owai-Seek) [reply code/modules/cargo/bounties/assistant.dm]: Unless you have some sort of admin or donor access that I don't access to, the only job that can spawn with "sunglasses" in the loadout are security, by spawning in with sec-huds. Otherwise, as far as I know, the QM and Bartender spawn with either sunglasses, or items that can be made into sunglasses.

File: code/modules/cargo/bounties/assistant.dm (hunk 7 above)
Comment (Owai-Seek) [reply code/modules/cargo/bounties/assistant.dm]: (As far as I know) The bounty can be completed by giving any kind of cuffs, not just handcuffs specifically. When giving out the bounty sheet, it will show as "Handcuffs", making people assume that only security handcuffs will fulfill it because they do not have access to the bounty description as cargo does.

File: code/modules/cargo/bounties/assistant.dm (hunk 6 above)
Comment (Owai-Seek) [reply code/modules/cargo/bounties/assistant.dm]: I mostly want to keep assistant bounties as assistant bounties, so I could move sunglasses to Security.

Comment (Ghommie) [reply code/modules/cargo/bounties/assistant.dm]: Guess I'll have to revert the description myself later, what an ordeal.

File: code/modules/cargo/bounties/assistant.dm (hunk 7 above)
Comment (Ghommie) [reply code/modules/cargo/bounties/assistant.dm]: Yes, I know that, but they are still, by definition, handcuffs. If you want to restrict them to security handcuffs, code it.

**Comment by Owai-Seek** [comment]
> what ghom said

I've fixed these then. Reference restored (but moved to "silly"), Cuff's reverted.

**Review by silicons — APPROVED** [review]
